credentials/build.gradle.kts

```diff
@@ -22,4 +22,5 @@ dependencies {
   implementation("decentralized-identity:did-common-java:1.9.0")
 
   testImplementation(kotlin("test"))
+  testImplementation("org.junit.jupiter:junit-jupiter:5.10.0")
```

Comment:
Given this is present in https://github.com/TBD54566975/web5-kt/blob/main/build.gradle.kts#L46

Curious why you're adding it here?